lms/static/js/verify_student/views/step_view.js

```diff
@@ -101,4 +103,4 @@
 
     });
 
- })( jQuery, _, _.str, Backbone, gettext );
+ })( jQuery, _, Backbone, gettext );
```

Comment:
Ok so the view definition is executed, and then by the time that the view is actually called into by test code `_.str` is defined? I find this concerning, though I understand that it may not be in scope for the underscore library upgrade itself. If you're familiar with how we currently handle shimming with require vs non-require JS, would you mind making the tech debt ticket? I'm having a hard time pinning down the exact work which would need to be done.